### ISSUE TITLE
Error in executing jar file

### ISSUE THREAD

**Issue by anandmr0** [jfoenixadmin/JFoenix#134]
hey i used JFoenix to develop an appliction but  i m not able to execute the .jar or exe file 

Exception in Application start method
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs
ncherImpl.java:389)
        at com.sun.javafx.application.LauncherImpl.launchApplication(Launche
l.java:328)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at sun.launcher.LauncherHelper$FXHelper.main(Unknown Source)
Caused by: java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(Launch
pl.java:917)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$
LauncherImpl.java:182)
        at java.lang.Thread.run(Unknown Source)
Caused by: javafx.fxml.LoadException:
file:/C:/Users/anand/Desktop/demo.jar!/demo/FXMLDocument.fxml

        at javafx.fxml.FXMLLoader.constructLoadException(FXMLLoader.java:260
        at javafx.fxml.FXMLLoader.importClass(FXMLLoader.java:2848)
        at javafx.fxml.FXMLLoader.processImport(FXMLLoader.java:2692)
        at javafx.fxml.FXMLLoader.processProcessingInstruction(FXMLLoader.ja
661)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2517)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2441)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:3214)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:3175)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:3148)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:3124)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:3104)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:3097)
        at demo.Demo.start(Demo.java:22)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1
(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(Pla
mImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformI
java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(Platf
mpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDisp
er.java:95)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplicatio
va:191)
        ... 1 more
Caused by: java.lang.ClassNotFoundException: com.jfoenix.controls.JFXButton
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at javafx.fxml.FXMLLoader.loadTypeForPackage(FXMLLoader.java:2916)
        at javafx.fxml.FXMLLoader.loadType(FXMLLoader.java:2905)
        at javafx.fxml.FXMLLoader.importClass(FXMLLoader.java:2846)
        ... 20 more
Exception running application demo.Demo

**Comment by alexsadliak** [comment]
I have the same problem, how can I solve it? I'll be grateful for any help.

**Comment by SergeyNik** [comment]
The solution to this problem here:
https://stackoverflow.com/questions/31189449/error-manifest-mf-already-exists-in-vfs-when-creating-new-artifact
"When creating the new artifact, I selected JAR and then a popup window displays default settings. It has me app module selected and the main class below is blank.
I was selecting the main class and it was producing an error. When I just accepted the default options left the main class blank, the artifact was created."
